image_client/botany_importer.py

```diff
@@ -59,7 +53,7 @@ def process_barcode(self, barcode, filepath_list):
             self.logger.debug(f"No barcode; skipping")
             return
         self.logger.debug(f"Barcode: {barcode}")
-        sql = f"select collectionobjectid  from collectionobject where catalognumber={barcode}"
+        sql = f"select CollectionObjectID from casbotany.collectionobject where CatalogNumber={barcode};"
```

Comment:
remove casbotany. prefix from these calls unless we need it - it makes the code more applicible to other collections if we end up re-using it.